### PR TITLE
Improve registration precess

### PR DESCRIPTION
**Pull request by vitalii-bulyzhyn**
PR includes:
- Change layout of the login page
- Change the layout of the creation user process
- Move setting password to the first step of the creation user
- Move setting password to the new user API
- Detect when login is expiring and handle this properly
- Fix tests

### CODE REVIEW

File: frontend/src/router/router.js [hunk 1]
```diff
@@ -748,11 +748,6 @@ const routerOptions = [
       title: "Login"
     })
   },
-  {
-    path: '/user/account/password/set',
```
Comment (anthony-sarkis) [inline frontend/src/router/router.js:752]: We would still want this (and the component) for an existing user to change their password right?

File: frontend/src/components/user/builder/builder_signup.vue [hunk 2]
```diff
@@ -317,7 +334,7 @@
             // done any other actions to refresh user information
             this.$store.commit('set_current_user', response.data.user)
 
-            this.route_set_password()
+            this.route_new_project()
```
Comment (anthony-sarkis) [inline frontend/src/components/user/builder/builder_signup.vue:337]: The builder sign up is the next step right?

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,101 +1,101 @@
...
-       WALRUS_SERVICE_URL_BASE: ${WALRUS_SERVICE_URL_BASE}
-       INTER_SERVICE_SECRET: ${INTER_SERVICE_SECRET}
-       PYTHONPATH: "/app/"
-       DIFFGRAM_INSTALL_FINGERPRINT: ${DIFFGRAM_INSTALL_FINGERPRINT}
-       DIFFGRAM_VERSION_TAG: ${DIFFGRAM_VERSION_TAG}
-       DIFFGRAM_HOST_OS: ${DIFFGRAM_HOST_OS}
-       DIFFGRAM_ERROR_SEND_TRACES_IN_RESPONSE: ${DIFFGRAM_ERROR_SEND_TRACES_IN_RESPONSE}
-       DIFFGRAM_SERVICE_NAME: ${DIFFGRAM_SERVICE_NAME}
-     entrypoint: ["python3", "-u", "/app/main.py"]
+  #  dispatcher:
```
Comment (anthony-sarkis) [inline docker-compose.yaml:3]: Is this a WIP that it's commented?

**Review by anthony-sarkis — COMMENTED** [review]
Thanks Vitalii! I just made some small notes

File: frontend/src/router/router.js (hunk 1 above)
Comment (vitalii-bulyzhyn) [reply frontend/src/router/router.js:752]: Hey Anthony! Yeah, I was fixing tests and noted

File: docker-compose.yaml (hunk 3 above)
Comment (vitalii-bulyzhyn) [reply docker-compose.yaml]: Yep, I will uncomment when the final version is ready

File: frontend/src/components/user/builder/builder_signup.vue (hunk 2 above)
Comment (vitalii-bulyzhyn) [reply frontend/src/components/user/builder/builder_signup.vue:337]: No, this is already builder signup

Comment (anthony-sarkis) [reply frontend/src/components/user/builder/builder_signup.vue:337]: oh ok I got order mixed up I think thanks

**Pull request merged**